Modules/ITS/src/ITSDecodingErrorCheck.cxx

```diff
@@ -30,14 +30,30 @@ namespace o2::quality_control_modules::its
 
 Quality ITSDecodingErrorCheck::check(std::map<std::string, std::shared_ptr<MonitorObject>>* moMap)
 {
+  // set timer
+  std::chrono::time_point<std::chrono::high_resolution_clock> start;
+  std::chrono::time_point<std::chrono::high_resolution_clock> end;
+  int TIME;
+  start = std::chrono::high_resolution_clock::now();
```

Comment:
I would create start 'start' timer in the initialization of the check ( `init ()` function). 

Idea is that we need to now the total time of the run, not just a time you spend in one `check` function

Summarizing: 

init() : create object `std::chrono::time_point<std::chrono::high_resolution_clock> start;`, start timer

 check(): measure how much time we have after initialization (it should roughly correspond to start of the run)